src/tests/unit/gna/gna_export_import_test.cpp

```diff
@@ -162,11 +164,38 @@ TEST_F(GNAExportImportTest, ExportImportI16) {
 }
 
 TEST_F(GNAExportImportTest, ExportImportI8) {
-    ov::AnyMap gna_config = {
+    gna_config = {
         ov::intel_gna::execution_mode(ov::intel_gna::ExecutionMode::SW_EXACT),
         ov::hint::inference_precision(ngraph::element::i8)
     };
     exported_file_name = "export_test.bin";
     ExportModel(exported_file_name);
     ImportModel(exported_file_name);
 }
+
+TEST_F(GNAExportImportTest, ShowLibVersionFromModelInLogDebugMode) {
+
+    gna_config = {
+        ov::log::level(ov::log::Level::DEBUG)
+    };
+    exported_file_name = "export_test.bin";
+    ExportModel(exported_file_name);
+    testing::internal::CaptureStdout();
+    ImportModel(exported_file_name);
+    
+    EXPECT_THAT(testing::internal::GetCapturedStdout(),
+                HasSubstr(ov::intel_gna::common::get_openvino_version_string()));
+}
+
+TEST_F(GNAExportImportTest, HideLibVersionFromModelInLogNoDebugMode) {
+    gna_config = {
+        ov::log::level(ov::log::Level::NO)
```

Comment:
It is better to test all the log levels
enum Level
{
    NO      = -1,
    ERR     = 0,
    WARNING = 1,
    INFO    = 2,
    DEBUG   = 3,
    TRACE   = 4,
};

An I think trace level should print the log too

Reply:
Could you please add the following tests:
1) Export model without the both versions the Import it
2) Export model without one of the versions or version should be empty an import it

Reply:
1. Adding tests for: NO, DEBUG and WARNING rest of modes is currently not supported in GNA plugin and model can't be created for such configuration.
2. As we talked lets treat it as separate task for creating test which validates backward compatibility for import